docs/samples/custom/grpc-server/client/client.go

```diff
@@ -0,0 +1,97 @@
+// +build grpcping
+
+package main
+
+import (
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"time"
+
+	pb "github.com/knative/docs/docs/serving/samples/grpc-ping-go/proto"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+var (
+	serverAddr         = flag.String("server_addr", "127.0.0.1:8080", "The server address in the format of host:port")
+	serverHostOverride = flag.String("server_host_override", "", "")
+	insecure           = flag.Bool("insecure", false, "Set to true to skip SSL validation")
+	skipVerify         = flag.Bool("skip_verify", false, "Set to true to skip server hostname verification in SSL validation")
+)
+
+func main() {
```

Comment:
Do you have a simple model serving gRPC example? this looks like the knative gRPC example :)

Reply:
yes, It is the knative grpc example. Is it neccesarry to change a example?

Reply:
I have updated the sample code to a simple helloworld server.